4001-5000/LIT4983ListDabra.xml

```diff
@@ -34,7 +34,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Genealogy of the abbots of <placeName ref="INS0341DL"/>.</p>
```

Comment:
It is **not** a genealogy, genealogy is about who monastically produced whom, it is very different from a list of abbots (echage)

Reply:
Yes, sorry for bothering you with so many mistakes!